dozer-ingestion/src/connectors/snowflake/tests.rs

```diff
@@ -48,9 +47,7 @@ fn connector_disabled_test_e2e_connect_snowflake_and_read_from_stream() {
         let op = iterator.write().next();
```

Comment:
@karolisg This test looks weird. I didn't change the behaviour but the old behaviour doesn't seem right.

Reply:
From @karolisg the test will hang if we can't get data from snowfake.